onnx/backend/test/case/node/deformconv.py

```diff
@@ -0,0 +1,129 @@
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+import onnx
+
+from ..base import Base
+from . import expect
+
+
+class DeformConv(Base):
```

Comment:
Could you please provide a better coverage of operator attributes? For example, `bias` has no test associated with it.

Reply:
Added some more cases.